Descent3/winmain.cpp

```diff
@@ -53,15 +54,15 @@ const char *French_strings[] = {
     "Votre processeur et votre systFme doivent prendre en charge Katmai pour exTcuter ce jeu."};
 
 const char *German_strings[] = {
-    "Descent3 unter Windows NT ben�tigt fnr die Ausfnhrung die NT-Version 4.0 oder h�her",
-    "Descent 3 ben�tigt fnr die Ausfnhrung Windows 9x, NT 4.0 oder h�her.",
+    "Descent3 unter Windows NT ben�tigt fnr die Ausfnhrung die NT-Version 4.0 oder h�her",
```

Comment:
revert encoding change

Reply:
I don't see that diff locally.

Reply:
your IDE changed encoding from WINDOWS-1252 to UTF-8

Reply:
We should decide what to do with these messages. Last time I mentioned that changing encoding to UTF-8 may broke messages on locales other than en, but now I think we can just convert these files into UTF-8 and deal with potential issues with non-English locales later.

Reply:
It would be a lot nicer to use GNU gettext or something so the languages lives in separate files, and not the source files.

Reply:
You can rebase onto current main, and this encoding mismatch will be gone.

Reply:
It turns out, it was vs code to blame here, I found it locally.